dev/com.ibm.ws.classloader.context/resources/com/ibm/ws/classloader/context/internal/resources/ClassLoaderContextMessages.nlsprops

```diff
@@ -0,0 +1,27 @@
+###############################################################################
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME com.ibm.ws.classloader.context/resources/com/ibm/ws/classloader/context/internal/resources/ClassLoaderContextMessages.nlsprops
+#ISMESSAGEFILE TRUE
+#NLS_ENCODING=UNICODE
+#
+#COMPONENTPREFIX CWWKL
+#COMPONENTNAMEFOR CWWKL Class Loader Context
+#
+# NLS_MESSAGEFORMAT_VAR
+#
+
+####################################################################################
+# NOTE: This message bundle has only reserved CWWKL0090-CWWKL0099 if more messages #
+#       are needed, reserve a new block of messages in the classloading bundle     #
+####################################################################################
+cannot.apply.classloader.context=CWWKL0090E: Unable to apply classloader context to an unmanaged thread. Ensure that work is being submitted to a thread pool that is managed by the server.
```

Comment:
'submitted to a thread pool' is too specific.  There are also APIs for running with context on any existing thread, which could lead to this same problem.  'Ensure that work is being submitted to a thread pool that is managed by the server' is too limited, given the expected use cases for ContextService/ThreadContext.  Let's be more specific and say "Unable to apply class loader context to unmanaged thread {0}." specifically calling out the ForkJoinPool thread that blocks setContextClassLoader.  And we can expand the second sentence by switching to: "Ensure that work runs on a thread that is managed by the server or allows the setContextClassLoader operation."